pkg/scheduler/framework/plugins/defaultpreemption/default_preemption.go

```diff
@@ -438,7 +440,9 @@ func SelectCandidate(candidates []Candidate) Candidate {
 	}
 
 	// We shouldn't reach here.
-	klog.Errorf("should not reach here, no candidate selected from %v.", candidates)
+	for i := range candidates {
+		klog.ErrorS(errors.New("Candidate is not selected"), "Should not reach here", "candidate", candidates[i])
+	}
```

Comment:
can't we do: `klog.ErrorS(errors.New("no candidate selected"), "should not reach here", "candidates", candidates)`

Reply:
Thanks for your advice, I'll fix it.

Reply:
can you do this one?

Reply:
I didn't understand it correctly yesterday. I have revised it.